lib/routific.rb

```diff
@@ -9,7 +9,14 @@
 
 # Main class of this gem
 class Routific
-  attr_reader :token, :visits, :fleet
+  Error           = Class.new(StandardError)
+  InvalidEndpoint = Class.new(Error)
+
+  ENDPOINTS = %i(vrp vrp-long pdp pdp-long)
+
+  @@endpoint = ENDPOINTS.first
```

Comment:
Please don't use class variables, use instance variables on the class instead :)